docs/JestObjectAPI.md

```diff
@@ -119,6 +119,92 @@ test('implementation created by jest.genMockFromModule', () => {
 });
 ```
 
+This is how `genMockFromModule` will mock the follwing data types:
+
+#### `Function`
+
+A new function will be created. The new function will have no formal parameters and when called will return `undefined`. This functionality also applies to `async functions`.
+
+#### `Class`
+
+A new class will be created. The interface of the original class is maintained however all of the class member functions will be mocked.
+
+#### `Object`
+
+Objects are deeply cloned. Their interfaces are maintained and their values are mocked.
+
+#### `Array`
+
+The original array is ignored and a new empty array is created.
+
+#### `String`
+
+A new copy of the original stirng is mocked.
+
+#### `Number`
+
+A new copy of the original number is mocked.
+
+Example:
+
+<!-- prettier-ignore -->
```

Comment:
?

Reply:
So for some reason prettier is adding these unwanted parens after the closing class bracket

![image](https://user-images.githubusercontent.com/2328448/54133656-b11ae100-43d3-11e9-986e-b09b17ccd489.png)

Reply:
If I remove `js` in the opening code block in the markdown file prettier does not insert the weird parenthesis. 🤔 

![image](https://user-images.githubusercontent.com/2328448/54169947-07b70800-4432-11e9-851d-30fd530ffb08.png)

Reply:
Oh, it's just adding the parense. `new class Bar{}` and `new class Bar{}()` behaves the same.

Should it be `new`ed?

I opened up https://github.com/prettier/prettier/issues/5964, not sure if it's a bug or not

Reply:
Nice, I added another comment that may or may not be related to the same issue.